src/services/LocalStorage.ts

```diff
@@ -0,0 +1,17 @@
+import {NotesType} from '../widget/notes/Notes';
+
+export const serviceLocalStorage = (
```

Comment:
Your function returns an object - wrong function name! Call it getNotesFromStorage, for example